src/OpenTelemetry.Instrumentation.Runtime/CHANGELOG.md

```diff
@@ -34,6 +34,9 @@
 
   This does not affect applications targeting .NET Framework.
 
+* Add "process.runtime.dotnet.gc.objects.size" metric
```

Comment:
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.Runtime#processruntimedotnetgcheapsize 
Could you update the readme file as well?

Reply:
would be also add a note about how is it different from the gc.heap.size (the fact that this exclude fragmentation, and works anytime even before the 1st GC event itself etc.) A lot such useful info was in this PR as comments. Its good to get the gist into the readme file itself to not lose them :)